workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Android
+{
+    public static class AndroidDeviceInfo
+    {
+        public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
+
+        private static readonly AndroidJavaObject Build = new AndroidJavaObject("android.os.Build");
```

Comment:
I have a few issues with these:
- should this object be disposed?
- it feels a bit dangerous to have these be statically evaluated (it'd be evaluated probably when you call IsAndroidStudioEmulator the first time)

If the values will never change and will never be used in other methods, I recommend to:
- remove the GetValue method
- move these into the method, which would:
    - get the java object
    - get the values
    - do the check
    - save into a bool
    - dispose of the java object
    - then reuse the bool when it may be queried again

Reply:
This looks like a reasonable change to me in any case. I'll add that.